### PR TITLE
dotCMS/core#23889 Permissions Tab Doesn't Load

### LINKED ISSUE #23889: Permissions tabs not loading
### Problem Statement

When the users try to load the permission tab in the portlet it throws an error in the browser console, and it doesn't load.

### Steps to Reproduce

1. Go to templates
2. Edit a design template
3. Click the permission tab
4. See the error

### Acceptance Criteria

User should be able to load the permission tab 

### dotCMS Version

master

### Proposed Objective

Code Maintenance

### Proposed Priority

Priority 1 - Show Stopper

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

image

### Assumptions & Initiation Needs

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* fixed permission tab load issue

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Comment by zulqarnainvd** [comment]
[screen-capture (14).webm](https://user-images.githubusercontent.com/113915849/223411197-c1656ec5-8531-4c6f-9341-29bb23fdfe43.webm)

**Comment by zulqarnainvd** [comment]
old

**Comment by zulqarnainvd** [comment]
[screen-capture (15).webm](https://user-images.githubusercontent.com/113915849/223411529-cf55f1a7-3ee5-4bff-a7d4-6ce221bb194a.webm)

**Comment by zulqarnainvd** [comment]
updated

File: dotCMS/src/main/webapp/html/portlet/ext/common/edit_permissions_tab_inc.jsp [hunk 1]
```diff
@@ -104,7 +104,7 @@
 </div>
 
 <!-- START Saving permissions dialog -->
-	<div id="savingPermissionsDialog" dojoType="dijit.Dialog" title="<%= LanguageUtil.get(pageContext, "saving-permissions") %>" style="display: none;">
+	<div id="savingPermissionsDialog" data-dojo-props="closable:false" dojoType="dijit.Dialog" title="<%= LanguageUtil.get(pageContext, "saving-permissions") %>" style="display: none;">
```
Comment (rjvelazco) [inline dotCMS/src/main/webapp/html/portlet/ext/common/edit_permissions_tab_inc.jsp:107]: Why do we set `data-dojo-props="closable:false"` and in the other `divs`?

File: dotCMS/src/main/webapp/html/portlet/ext/common/edit_permissions_tab_js_inc.jsp [hunk 2]
```diff
@@ -113,20 +113,17 @@
...
     var permissionsLoaded = false;
-	//Initialization
-	dojo.addOnLoad(preloadingPermissions);
 
-	require(['dojo/_base/window', 'dojo/on'], function(baseWin, on) {
-		on(baseWin.global, 'load', function() {
-			loadPermissions();
+	require(['dojo/ready'], function(ready){
+		ready(function(){
+			preloadingPermissions();
```
Comment (rjvelazco) [inline dotCMS/src/main/webapp/html/portlet/ext/common/edit_permissions_tab_js_inc.jsp:126]: Nice! This is more readable!

**Review by rjvelazco — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/webapp/html/portlet/ext/common/edit_permissions_tab_inc.jsp (hunk 1 above)
Comment (zulqarnainvd) [reply dotCMS/src/main/webapp/html/portlet/ext/common/edit_permissions_tab_inc.jsp:107]: previously we are hiding dialog close button using javascript but in  dojo dialog have prop `data-dojo-props="closable:false"` for hiding the close button.

**Review by alfredo-dotcms — APPROVED** [review]

**Pull request merged**